### DISCUSSION TITLE
Cursor out of window bounds behavior

### DISCUSSION THREAD

**Discussion by MichalLebeda** [hecrj/iced#1472] [General]
Currently```Cursor``` value is ```Cursor::Unavailable``` if cursor is less than window origin. This behavior is not consistent as `x` and `y` values can be bigger than window origin + size , thus escaping window (which is fine).
Why this matters: sometimes user can scroll by dragging on widget or taking similar action. Moving out of the window right or down is fine but not up or left and it does not make any sense to me as sometimes I can move (for example) in a map or similar widget. 
**I think that this is a really important topic for further usecases**

**Comment by 13r0ck** [comment]
This may be better as an issue, as it sounds like a bug. Please also include a [Simple Short Self-Contained, Correct Example](ssccee.org).

**Reply by MichalLebeda** [reply]
link is dead, reported it as follows:[https://github.com/iced-rs/iced/issues/1481]( https://github.com/iced-rs/iced/issues/1481)
